step-definitions/Hotels/Tc-22.js

```diff
@@ -0,0 +1,80 @@
+//TC22
+
+const { Given, Then, When, And } = require("@wdio/cucumber-framework")
+const {expect} = require("chai");
+
+
+Given(/^I am on hotels landing page$/, async function () {
+    await browser.url('https://www.hotels.com/')
```

Comment:
Once you implement this step you don't need to implement the same step again. Just reuse it.